### PR TITLE
Restore password field

### PR DESCRIPTION
**Pull request by gabeobrien**

### CODE REVIEW

**Review by Julien4218 — APPROVED** [review]
Looks good

File: test/deploy/linux/haproxy/install/debian/roles/configure/tasks/main.yml [hunk 1]
```diff
@@ -33,9 +33,9 @@
 
 - block:
   - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=newrelic >> ~/.bashrc"
-  - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=Virtuoso4all! >> ~/.bashrc"
+    shell: "echo export NR_CLI_USERNAME=newrelic >> ~/.bashrc"
```
Comment (gabeobrien) [inline test/deploy/linux/haproxy/install/debian/roles/configure/tasks/main.yml:36]: This line need to be updated

File: test/deploy/linux/haproxy/install/rhel/roles/configure/tasks/main.yml [hunk 2]
```diff
@@ -33,9 +33,9 @@
 
 - block:
   - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=newrelic >> ~/.bashrc"
-  - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=Virtuoso4all! >> ~/.bashrc"
+    shell: "echo export NR_CLI_USERNAME=newrelic >> ~/.bashrc"
```
Comment (gabeobrien) [inline test/deploy/linux/haproxy/install/rhel/roles/configure/tasks/main.yml:36]: and here again

File: test/deploy/linux/rabbitmq/install/debian/roles/configure/tasks/main.yml [hunk 3]
```diff
@@ -43,9 +43,9 @@
 
 - block:
   - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=newrelic >> ~/.bashrc"
-  - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=Virtuoso4all! >> ~/.bashrc"
+    shell: "echo export NR_CLI_USERNAME=newrelic >> ~/.bashrc"
```
Comment (gabeobrien) [inline test/deploy/linux/rabbitmq/install/debian/roles/configure/tasks/main.yml:46]: here as well

File: test/deploy/linux/rabbitmq/install/rhel/roles/configure/tasks/main.yml [hunk 4]
```diff
@@ -83,9 +83,9 @@
 
 - block:
   - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=newrelic >> ~/.bashrc"
-  - name: Export NR_CLI_DB_USERNAME
-    shell: "echo export NR_CLI_DB_USERNAME=Virtuoso4all! >> ~/.bashrc"
+    shell: "echo export NR_CLI_USERNAME=newrelic >> ~/.bashrc"
```
Comment (gabeobrien) [inline test/deploy/linux/rabbitmq/install/rhel/roles/configure/tasks/main.yml:86]: again

**Review by Julien4218 — APPROVED** [review]

**Review by Julien4218 — APPROVED** [review]

**Pull request merged**